### ISSUE TITLE
Consider adding support for use latest

### ISSUE THREAD

**Issue by TomasHubelbauer** [coreybutler/nvm-windows#440]
I have a feature suggestion I'd like to open for discussion: to enable on to use `nvm use latest` similarly to how `install` can accept `latest` in place of a version name.

The use case here is that when switching between Node versions because that one project just requires 9 but all your other projects keep up with latest, you have to remember what is the latest version you have every time when switching from 9 or other old version you need to keep using.

I think `use latest` could work in two ways: either pick the latest version of the ones you have locally installed (which would probably be very simple to implement and IMO still semantically valid and useful) or `install latest` immediately followed by `use` in one command. I think the former is much simpler and personally it's all I need but I am not sure if the use of the word `latest` there might be deemed too confusing/surprising?

Thank you for considering this feature request.

**Comment by WORMSS** [comment]
I too was caught out by this.. 
```sh
> nvm install latest
Downloading node.js version 16.1.0 (64-bit)...
Complete

>nvm use latest
latest.0.0
node vlatest.0.0 (64-bit) is not installed.
FAILED!!!
```

**Comment by coreybutler** [comment]
This has been, but the keyword is `newest`. 

The following commands are now supported:

- `nvm use newest` uses the newest version _currently installed_.
- `nvm use latest` attempts to use the Latest release (as found on nodejs.org)
- `nvm use lts` attemtps to use the LTS release (as found on nodejs.org)

This will be available in v1.1.8.
